README.rst

```diff
@@ -119,36 +132,156 @@ a minimum of 2 CPUs and 6GB of memory works well.
    the services directly via Django admin at the ``/admin/`` path, or login via
    single sign-on at ``/login/``.
 
-   Default:
+.. code:: sh
 
-   .. code:: sh
+    make dev.provision
 
-       make dev.provision
 
-   Provision using `docker-sync`_:
+8. Start the services. This command will mount the repositories under the
+   DEVSTACK\_WORKSPACE directory.
 
-   .. code:: sh
+   **NOTE:** it may take up to 60 seconds for the LMS to start, even after the ``make dev.up`` command outputs ``done``.
 
-       make dev.sync.provision
+.. code:: sh
 
+    make dev.up
 
-5. Start the services. This command will mount the repositories under the
-   DEVSTACK\_WORKSPACE directory.
 
-   **NOTE:** it may take up to 60 seconds for the LMS to start, even after the ``make dev.up`` command outputs ``done``.
+Wordpress Setup
+---------------
 
-   Default:
+Gulp should be installed before proceeding further (you may need to skip sudo if you are using nvm).
 
-   .. code:: sh
+.. code:: sh
 
-       make dev.up
+    sudo npm i -g gulp-cli
 
-   Start using `docker-sync`_:
+1. Install php and composer in host machine as well as wordpress container.
 
-   .. code:: sh
+    For host machine (Ubuntu)
+
+.. code:: sh
+    sudo apt-get install php7.2
+    curl -s https://getcomposer.org/installer | php
+    sudo mv composer.phar /usr/local/bin/composer
+
+    For wordpress container
+
+.. code:: sh
+    make wordpress-shell
+    apt-get install php7.2
+    curl -s https://getcomposer.org/installer | php
+    mv composer.phar /usr/bin/composer
+
+2. Install the requirements for ``edly-wp-theme`` and ``edly-wp-plugin`` inside wordpress shell.
+
+.. code:: sh
 
-       make dev.sync.up
+    cd /var/www/html/wp-content/plugins/edly-wp-plugin
+    composer install
 
+    cd /var/www/html/wp-content/themes/edly-wp-theme
+    composer install
+
+3. Change permissions inside docker container.
+
+.. code:: sh
+
+    chmod 0777 /var/www/html/wp-content -R
```

Comment:
I believe we don't need this much open permissions. Try using following command:
```chown -R www-data:www-data wp-content```